packages/kit/src/core/prerender/crawl.js

```diff
@@ -160,6 +160,8 @@ export function crawl(html) {
 								href = value;
 							} else if (name === 'id') {
 								ids.push(value);
+							} else if (name === 'name') {
+								ids.push(value);
```

Comment:
i think this should do the trick?

```suggestion
								if (tag === 'A') ids.push(value);
```

Reply:
We should still check that `name === 'name'` as well, no?

Reply:
Added the `tag === 'A'` check